communication/moderators.md

```diff
@@ -33,14 +49,36 @@ See our [moderation guidelines](./moderating.md) for policies and recommendation
 - Bob Killen (@mrbobbytables) - ET
 - Jeffrey Sica (@jeefy) - ET
 
-### Regional category moderators
+Note, due to how the software is designed, dedicated moderators are not required
+for discuss, the [trust
+system](https://blog.discourse.org/2018/06/understanding-discourse-trust-levels/)
+in place allows for self moderation. In this case administrators act as
+exception handlers. Regional moderators are required for language/cultural
+reasons, (ie. we can't expect a community member who doesn't speak a language to
+be able to effectively moderate). 
 
-- [Chinese]
-- [German]
-- [Italian]
-- [Ukrainian]
+### Regional category moderators
 
-## YouTube Channel
+- [Chinese] (3)
+    - Open
+    - Open
+    - Open
+- [German] (3)
+    - Open
+    - Open
+    - Open
+- [Italian] (3)
+    - Open
+    - Open
+    - Open
+- [Ukrainian] (3)
+    - Open
+    - Open
+    - Open
+
+## YouTube Channel (9)
+
+### Primary Moderators
 
 - Paris Pittman (@parispittman) - PT
 - Sarah Novotny (@sarahnovotny) - PT
```

Comment:
Let’s ask Sarah if she still wants to be here. Also remove bob. Maybe make joe an inactive admin since he doesn’t mod but still keep listed since he is on the account?

Reply:
Sarah's listed as the primary, I've asked her over mail to transfer, we can then make most of us "communications managers", that will allow uploads and maintenance without too much admin power.